source/contributing/guidelines.rst

```diff
@@ -34,155 +40,5 @@ General steps
-     * This file is part of Sponge, licensed under the MIT License (MIT).
-     *
-     * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
-     * Copyright (c) contributors
-     *
-     * Permission is hereby granted, free of charge, to any person obtaining a copy
-     * of this software and associated documentation files (the "Software"), to deal
-     * in the Software without restriction, including without limitation the rights
-     * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-     * copies of the Software, and to permit persons to whom the Software is
-     * furnished to do so, subject to the following conditions:
-     *
-     * The above copyright notice and this permission notice shall be included in
-     * all copies or substantial portions of the Software.
-     *
-     * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-     * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-     * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-     * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-     * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-     * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-     * THE SOFTWARE.
-     */
-
-    package com.example.java;
-
-    import com.google.common.base.Optional;
-    import org.slf4j.Logger;
-    import org.slf4j.LoggerFactory;
-
-    import java.util.Random;
-
-    public class Example {
-
-        private static final Logger log = LoggerFactory.getLogger(Example.class);
-        private static final Random random = new Random();
-        private final String id = "test";
-
-        /**
-         * Returns an identifier approximately half of the time.
-         *
-         * <p>A static instance of {@link Random} is used to calculate the
-         * outcome with a 50% chance.</p>
-         *
-         * @return The ID, if available
-         */
-        public Optional<String> resolveId() {
-            log.info("ID requested");
-
-            if (random.nextBoolean()) {
-                return Optional.of(this.id);
-            } else {
-                return Optional.empty();
-            }
-        }
-
-        /**
-         * Returns an identifier approximately half of the time.
-         *
-         * <p>A static instance of {@link Random} is used to calculate the
-         * outcome with a 50% chance. If the outcome is to not return the ID,
-         * the given fallback ID is returned.</p>
-         *
-         * @param fallback A fallback name to return
-         * @return The ID half of the time, the given fallback the other half
-         */
-        public String resolveId(String fallback) {
-            return resolveId().or(fallback);
-        }
-
-    }
+#. When your PR gets merged, we'll make sure that it's rebased on ``master`` branch and squashed, if you didn't do this
```

Comment:
sure that people should rebase on master? with the new model, master is only for releases I think. this may be project-dependant